src/viadot/sources/epicor.py

```diff
@@ -197,7 +197,7 @@ def parse_orders_xml(xml_data: str) -> pd.DataFrame:  # noqa: C901, PLR0912
                 line_item = LineItemDetail(**item_params_dict)
                 row = Order(HeaderInformation=header_info, LineItemDetail=line_item)
                 my_dict = row.dict()
-                final_df = final_df.append(
+                final_df = final_df._append(
```

Comment:
Not great to rely on a private method as it could be deprecated without warning. Maybe you can keep a list of dfs and use `pd.concat(dfs)` at the end of the loop?